libs/component-sample/src/lib/feature/search-result-list-sample/search-result-list-sample-card-data.component.html

```diff
@@ -0,0 +1,279 @@
+<div class="sds-card sds-card--vertical">
```

Comment:
Need to work with Alejandro or Sam to resolve font size, font weight, etc inadvertently flowing through to the elements within the card.

Example: spans within the card are getting 700 font weight